app/views/smart_proxies/_form.html.erb

```diff
@@ -12,8 +12,8 @@
   </ul>
   <div class="tab-content">
     <div class="tab-pane active" id="primary">
-      <%= text_f(f, :name) %>
-      <%= text_f(f, :url, :size => "col-md-8" ) %>
+      <%= text_f(f, :name, :help_inline => _("Must be unique")) %>
+      <%= text_f(f, :url, :label => _("URL"), :help_inline => _("HTTPS endpoint"), :label_help => _("Hostname and client certificate from Administer - Settings - Authentication must be valid. Ports are typically 8448 or 9090."), :size => "col-md-8" ) %>
```

Comment:
I would also suggest putting an actual link, instead of `Administer - Settings - Authentication`. Less prone to changes and easier for the user to understand.

Reply:
I wish I could, could not figure it out how to create a link to setting tab so it directly open. I simply removed this reference, that's it.